main.js

```diff
@@ -128,12 +128,25 @@ define(function (require, exports, module) {
             codeMirror.focus();
             CommandManager.execute(Commands[msgObj.command]);
         }
-        else if (msgObj.commandCategory === "viewCommand") {
-            ViewCommand[msgObj.command](msgObj.params);
+        else if (msgObj.commandCategory === "fontChange") {
+            CommandManager.execute(Commands[msgObj.Command]);
+            if(msgObj.params < "12") {
+                for(var i = 12; i > msgObj.params; i--) {
+                    CommandManager.execute(Commands["VIEW_DECREASE_FONT_SIZE"]);
+                }
+            }
+            else if(msgObj.params > "12") {
+                for(var i = 12; i < msgObj.params; i++) {
+                    CommandManager.execute(Commands["VIEW_INCREASE_FONT_SIZE"]);
+                }
+            }
         }
         else if (msgObj.commandCategory === "editorCommand") {
             Editor[msgObj.command](msgObj.params);
         }
+        else if (msgObj.commandCategory === "reloadCommand") {
+            // Klevers reload function call will go here
```

Comment:
This is a WIP. @K-LV has a patch that will expose the reload function that we can call here.